src/ispc.cpp

```diff
@@ -1334,10 +1334,13 @@ std::string Target::GetTripleString() const {
     case TargetOS::macos:
         // asserts
         if (m_arch != Arch::x86_64) {
-            Error(SourcePos(), "macOS target supports only x86_64.");
+            triple.setArchName("amd64");
+        } else if (m_arch == Arch::aarch64) {
+            triple.setArchName("arm64");
```

Comment:
`arm64` is also an alias for `aarch64`, though LLVM differentiates between arm64 and aarch64 targets. I'm not fully following this logic, but I agree that in this context `arm64` is the right naming here.

Reply:
I wasn't quite sure which to pick here, I saw that iOS used arm64 as the string but it looks like aarch64 also works fine so I can change this to the preferred string